responsibleai/responsibleai/_managers/explainer_manager.py

```diff
@@ -155,6 +160,109 @@ def list(self):
             props[Keys.IS_COMPUTED] = False
         return props
 
+    def get_data(self):
+        """Get explanation data
+
+        :return: A array of ModelExplanationData.
+        :rtype: List[ModelExplanationData]
+        """
+        return [
+            self._get_interpret(i) for i in self.get()]
+
+    def _get_interpret(self, explanation):
```

Comment:
In order to make this more unit testable, what do you think about introducing an `explainer_serializer.py` with an `ExplainerSerializer` object that will encapsulate the conversion from explanation to JSON-serializable data? @imatiach-msft @xuke444

Reply:
will use a separate pr for this.